src/strings/en-us.json

```diff
@@ -1100,8 +1100,8 @@
     "No": "No",
     "NoCreatedLibraries": "Seems like you haven't created any libraries yet. {0}Would you like to create one now?{1}",
     "NoNewDevicesFound": "No new devices found. To add a new tuner, close this dialog and enter the device information manually.",
-    "NoNextUpItemsMessage": "None found. Start watching your shows!",
-    "NoPluginConfigurationMessage": "This plugin has no settings to configure.",
+    "MessageNoNextUpItems": "None found. Start watching your shows!",
+    "MessageNoPluginConfiguration": "This plugin has no settings to configure.",
```

Comment:
Why rename this one and not the surrounding ones?

Feels like all the placeholders should have a unified name.

Reply:
I was specifically only renaming keys here that incorrectly used the placeholders.